www/pages/rapidsMap.php

```diff
@@ -1,145 +1,353 @@
+		
+	},
+    {
+        "title": 'Spanish-American War',
+        "lat": '43.129562',
+        "lng": '-77.639158',
+        "description":  '<img src=images/download.jpg style=width:100px;height:100px;/></br><h4>Robert Thomas</br><h6>(1829 - 1879)</h6></h4>Son of William Thomas,first Candlestick marker in Rochester,Robert was succeeded by his son Charles and help',
+		"desc": '<img src=images/download.jpg style=width:100px;height:100px;/></br><h4>Robert Thomas</br><h6>(1829 - 1879)</h6></h4>Son of William Thomas,first Candlestick marker in Rochester,Robert was succeeded by his son Charles and help'
+		
+   },
+    {
+        "title": 'Rapids Flora',
+        "lat": '43.129388',
+        "lng": '-77.639036',
+        "description":  '<img src=images/download.jpg style=width:100px;height:100px;/></br><h4>Robert Thomas</br><h6>(1829 - 1879)</h6></h4>Son of William Thomas,first Candlestick marker in Rochester,Robert was succeeded by his son Charles and help',
+		"desc": '<img src=images/download.jpg style=width:100px;height:100px;/></br><h4>Robert Thomas</br><h6>(1829 - 1879)</h6></h4>Son of William Thomas,first Candlestick marker in Rochester,Robert was succeeded by his son Charles and help'
+		
+	}
+    ];
+        window.onload = function () {
+            LoadMap();
+        };
+
+        var map;
+        var marker;
+        function LoadMap() {
+            var mapOptions = {
+                center: new google.maps.LatLng(markers[0].lat, markers[0].lng),
+                zoom: 20,
+				mapTypeId: 'satellite'
+            };
+            map = new google.maps.Map(document.getElementById("dvMap"), mapOptions);
+           /* SetMarker(0);*/
+        };
+        function SetMarker(position) {
+                if (marker != null) {
+                marker.setMap(null);
+            }
+			var data = markers[position];
+            var myLatlng = new google.maps.LatLng(data.lat, data.lng);
+            marker = new google.maps.Marker({
+                position: myLatlng,
+                map: map,
+                title: data.title
+            });
+			var infoWindow = new google.maps.InfoWindow();
+            infoWindow.setContent("<div><div class='first' style = 'width:250px;height:auto;text-align:center'>" + data.description + "</br></br><button onclick='openup()' class='btn' style='border-radius:25px;color: #ec5e07;background-color: #fff;border-color: #ec5e07;padding:5px !important;'>Learn More</button></div><div class='popup-overlay' style='display:none;'><div class='popup-content'>" + data.desc +"</br><button onclick='shutdown()' class='btn' style='border-radius:25px;color: #ec5e07;background-color: #fff;border-color: #ec5e07;padding:5px !important; margin-top: 15px;'>Return To Map</button></div></div></div>");

```

Comment:
If you know you are going to have code like this get into the habit of add a picture of the output somewhere. Probably the google drive and then link it in your PR

Reply:
Is this suppose to show the data from the link as well?